dashboards/mssql-server/alerts/connections-near-limit.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
Can you create a new top-level folder in this repo for alerts?
e.g.
/alerts/mssql-server/\*.json
instead of
/dashboards/mssql-server/alerts/\*.json